src/store/action.tsx

```diff
@@ -0,0 +1,57 @@
+import { ActionType } from "./actionTypes";
+
```

Comment:
I'd either move this types in actionCreators file or consider a different name, cause current name doesn't represent the content of this file